### PR TITLE
Update Rust.php

### PR DESCRIPTION
**Pull request by hej-julian**
Server Port + 1 = Query Port

### CODE REVIEW

File: src/GameQ/Protocols/Rust.php [hunk 1]
```diff
@@ -43,22 +43,11 @@ class Rust extends Source
...
     
-    /**
-     * Overload so we can get max players from mp of keywords and num players from cp keyword
+	/**
+     * query_port = client_port + 1
      *
-     * @param Buffer $buffer
+     * @type int
      */
-    protected function processDetails(Buffer $buffer)
```
Comment (Krymonota) [inline src/GameQ/Protocols/Rust.php:51]: What's the reason for removing this function?

Comment (hej-julian) [reply src/GameQ/Protocols/Rust.php:51]: With the function I have no online status, maybe you can test it too?

Comment (Austinb) [reply src/GameQ/Protocols/Rust.php:51]: It looks like Rust follows the regular source response for player information now.  Without the `processDetails` function both `max_players` and `num_players` have values.  Was able to find a server to test on that responded at `95.214.180.237:28067` using query_port+1.

**Comment by Austinb** [comment]
Assuming the conversation around `processDetails` is satisfied, code style needs to be fixed so tests will pass.

**Pull request closed without merging**